n_t_with_s/test.py

```diff
@@ -0,0 +1,246 @@
+
+"""
+Snakefile for running N_t analysis on stdpopsim.
+
+Simply running `snakemake` will run all analysis 
+defined by the arguments above.
+
+Currently, for each rep,
+This will run stairway plot, smc++, and msmc
```

Comment:
I'm confused about this file-- what's it doing?

Reply:
I was just using this to do some small tests, but excluding it now.